### PR TITLE
Change the netperf test version and source patch

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1934952

Signed-off-by: Lei Yang leiayang@redhat.com

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Hi @PaulYuuu Could you help me review it ? Thanks a lot.

**Comment by PaulYuuu** [comment]
`migrate.cfg` was deleted by mistake.

**Comment by yanglei-rh** [comment]
> `migrate.cfg` was deleted by mistake.

I have solved this issue, please help me review it again, thanks.

**Review by PaulYuuu — APPROVED** [review]

File: qemu/tests/cfg/netkvm_in_use.cfg [hunk 1]
```diff
@@ -10,7 +10,7 @@
     driver_name = "netkvm"
     run_bgstress = netperf_stress
     bg_stress_run_flag = netperf_run
-    netperf_link = netperf-2.6.0.tar.bz2
+    netperf_link = netperf-2.7.1.tar.bz2
```
Comment (PaulYuuu) [inline qemu/tests/cfg/netkvm_in_use.cfg:13]: It seems that windows only has `deps/netperf/netperf.exe` (2.6.0), I am not sure 2.7.1 can work with it together, please help confirm this one.

**Comment by PaulYuuu** [comment]
If possible, it would be better to delete the unused netperf source tar(`generic/deps/performance`)

File: qemu/tests/cfg/ovs_qos.cfg [hunk 2]
```diff
@@ -6,7 +6,7 @@
     netperf_server = vm2
     vms = ${netperf_client} ${netperf_server}
     netperf_client_options = " -l 60"
-    netperf_link = netperf-2.6.0.tar.bz2
+    netperf_link = netperf-2.7.1.tar.bz2
```
Comment (PaulYuuu) [inline qemu/tests/cfg/ovs_qos.cfg:9]: Remove the space at EOL.

File: qemu/tests/cfg/netkvm_in_use.cfg (hunk 1 above)
Comment (yanglei-rh) [reply qemu/tests/cfg/netkvm_in_use.cfg:13]: @PaulYuuu  It test pass.

**Comment by yanglei-rh** [comment]
> If possible, it would be better to delete the unused netperf source tar(`generic/deps/performance`)

Hi @PaulYuuu  I am not sure if this will affect others.

File: qemu/tests/cfg/ovs_qos.cfg (hunk 2 above)
Comment (yanglei-rh) [reply qemu/tests/cfg/ovs_qos.cfg]: I have removed it.

**Comment by PaulYuuu** [comment]
> > If possible, it would be better to delete the unused netperf source tar(`generic/deps/performance`)
> 
> Hi @PaulYuuu I am not sure if this will affect others.

As discussed at #2629, there are no tests to use them. However, we should not maintain too many netperf versions, so this step should be done now or in the future.

You can also submit another PR to fix it, it's up to you.

**Comment by yanglei-rh** [comment]
> > > If possible, it would be better to delete the unused netperf source tar(`generic/deps/performance`)
> > 
> > 
> > Hi @PaulYuuu I am not sure if this will affect others.
> 
> As discussed at #2629, there are no tests to use them. However, we should not maintain too many netperf versions, so this step should be done now or in the future.
> 
> You can also submit another PR to fix it, it's up to you.

Hi @PaulYuuu I think I should  sent a new patch to fix this problem, this is more targeted. And I need to check with performance colleagues on the path they use.

**Comment by PaulYuuu** [comment]
> > > > If possible, it would be better to delete the unused netperf source tar(`generic/deps/performance`)
> > > 
> > > 
> > > Hi @PaulYuuu I am not sure if this will affect others.
> > 
> > 
> > As discussed at #2629, there are no tests to use them. However, we should not maintain too many netperf versions, so this step should be done now or in the future.
> > You can also submit another PR to fix it, it's up to you.
> 
> Hi @PaulYuuu I think I should sent a new patch to fix this problem, this is more targeted. And I need to check with performance colleagues on the path they use.

Okay, then I have no other questions, please provide test results, thanks.

**Comment by yanglei-rh** [comment]
Test Pass(except rhel bug)
 (01/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.flow_caches_stress_test.multi_queues.q35: PASS (891.60 s)
 (02/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.flow_caches_stress_test.no_multi_queues.q35: PASS (890.79 s)
 (03/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtual_nic.send_buffer.default_buf.q35: PASS (257.75 s)
 (04/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_STREAM.guest2guest.q35: PASS (149.38 s)
 (05/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_STREAM.host2guest.q35: PASS (216.87 s)
 (06/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_RR.guest2guest.q35: PASS (152.43 s)
 (07/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_RR.host2guest.q35: PASS (217.56 s)
 (08/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_CRR.guest2guest.q35: PASS (149.69 s)
 (09/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_CRR.host2guest.q35: PASS (216.83 s)
 (10/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_MAERTS.guest2guest.q35: PASS (150.40 s)
 (11/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_MAERTS.host2guest.q35: PASS (217.12 s)
 (12/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_SENDFILE.guest2guest.q35: PASS (151.82 s)
 (13/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.TCP_SENDFILE.host2guest.q35: PASS (215.66 s)
 (14/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.UDP_STREAM.guest2guest.q35: PASS (146.09 s)
 (15/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.UDP_STREAM.host2guest.q35: PASS (215.68 s)
 (16/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.UDP_RR.guest2guest.q35: PASS (149.14 s)
 (17/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.netperf_stress_test.UDP_RR.host2guest.q35: PASS (216.22 s)
 (18/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.nic_opt.opt_test_netperf.indirect_desc.opt_on.q35: PASS (290.57 s)
 (19/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.nic_opt.opt_test_netperf.indirect_desc.opt_off.q35: PASS (288.68 s)
 (20/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.nic_opt.opt_test_netperf.mrg_rxbuf.opt_on.q35: PASS (292.30 s)
 (21/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.nic_opt.opt_test_netperf.mrg_rxbuf.opt_off.q35: PASS (285.55 s)
 (22/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.boot_with_different_vectors.with_vectors.q35: FAIL: Guest do not support msi when vectors = 0. (49.38 s)
 (23/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.boot_with_different_vectors.without_vectors.q35: PASS (167.20 s)
 (24/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.boot_with_different_vectors.with_vhostforce.q35: PASS (350.66 s)
 (25/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.boot_with_different_vectors.without_vhostforce.q35: PASS (354.25 s)
 (26/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.bridge_vlan.q35: PASS (537.39 s)
 (27/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.mq_change_qnum.disable_enable_queues.under_netperf_and_ping.enable_msi.q35: PASS (1159.50 s)
 (28/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.mq_change_qnum.disable_enable_queues.under_netperf_and_ping.disable_msi.q35: PASS (1203.22 s)
 (29/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp1.q35: FAIL: Run 1 netperf session, but 2 queues works (225.76 s)
 (30/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.single_netperf_session.q35: FAIL: Run 1 netperf session, but 3 queues works (206.58 s)
 (31/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.smp4.multiple_netperf_session.q35: PASS (268.80 s)
 (32/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.max_queues.q35: FAIL: Run 2 netperf session, but 4 queues works (212.66 s)
 (33/33) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_queues_test.with_netperf.cpu_affinity.q35: PASS (274.62 s)
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.ovs_qos.q35: PASS (264.53 s)

**Pull request merged**